tasks/main.yml

```diff
@@ -81,12 +81,13 @@
 - name: get the current default route
   command: "/usr/sbin/ip route show 0.0.0.0/0"
   register: default_route
+  when: not ansible_check_mode
```

Comment:
Where/how is that defined ?

Reply:
oh, probably set when ansible-playbook is kicked with `--check` ? and so ansible_check_mode: False when not called with it ?